### ISSUE TITLE
Favourites 

### ISSUE THREAD

**Issue by MatheMann1** [Binabh/based-cooking-app#5]
It would be very nice to mark recipes as favourites.

**Comment by CharanSatwik** [comment]
Can I try working on this?

**Comment by Binabh** [comment]
@CharanSatwik Yes you may work on this. I will review if you create a PR

**Comment by CharanSatwik** [comment]
> @CharanSatwik Yes you may work on this. I will review if you create a PR

Thank you, I'll see what I can do

**Comment by CharanSatwik** [comment]
Hey, I hit a build failure with current Flutter stable (3.41.x)  the Gradle files use the old imperative `apply plugin` style, which newer Flutter versions no longer support (needs the declarative `plugins {}` block). Would you be open to a PR that migrates the Android Gradle setup to the modern format?

**Comment by Binabh** [comment]
@CharanSatwik Yes, The project is very old. Yes I will happily accept a PR for upgrade of tooling and dependencies.

**Comment by CharanSatwik** [comment]
> @CharanSatwik Yes, The project is very old. Yes I will happily accept a PR for upgrade of tooling and dependencies.

Thank you very much.
